probe-rs/src/probe/stlink/mod.rs

```diff
@@ -281,6 +281,9 @@ impl<D: StLinkUsb> STLink<D> {
     /// Minimum required STLink firmware version.
     const MIN_JTAG_VERSION: u8 = 26;
 
+    /// Minimum required STLink V3 firmware version.
```

Comment:
Could you maybe add a comment here that this barrier was introduced because the ST-Link locks up? :) So we will know in the future when people come asking why we don't support older firmwares (had that already).

Reply:
Added :) Check if it's good enough please.

Reply:
Very well done :)